zuul/connection/github.py

```diff
@@ -559,7 +559,7 @@ def setCommitStatus(self, owner, project, sha, state,
 
     def labelPull(self, owner, project, pr_number, label):
         pull_request = self.github.issue(owner, project, pr_number)
-        pull_request.add_label(label)
+        pull_request.add_labels(label)
         log_rate_limit(self.log, self.github)
```

Comment:
This is confusing. I went to check on the method and realized that the object is an issue even though the variable name is pull_request. 

You're still right that the method name is wrong but we should rename pull_request to issue here.